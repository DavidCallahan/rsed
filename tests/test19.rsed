$text = "foo $ bar"
if $text =~ " $ "r then
   print "match 1"
end
if $text =~ " \\\\$" then
   print "match 2"
end
$text = "foo bar"
if $text =~ " $ "r then
   print "match 3"
end
