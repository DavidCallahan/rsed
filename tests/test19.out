match 1
match 1x
match 2
not match 3
not match 3x
foo $\.*+?()[]{} bar
match 4
match 4x
aXbXc
a$b$X
